assets/package.json

```diff
@@ -54,6 +54,7 @@
     "postcss-import": "^14.0.0",
     "postcss-loader": "^6.1.1",
     "prettier": "^2.2.1",
+    "tailwind-scrollbar": "^1.3.1",
```

Comment:
We have a `.tiny-scrollbar` class that we use in a couple places :)

Reply:
fixed in 06800fe